docs/cloud/managing-airbyte-cloud.md

```diff
@@ -186,6 +186,16 @@ These parameters apply to all streams in the connection.
 
 :::
 
+If you need to use [cron scheduling](http://www.quartz-scheduler.org/documentation/quartz-2.3.0/tutorials/crontrigger.html):
+1. In the **Replication Frequency** dropdown, click **Cron**. 
+2. Enter a cron expression and choose a time zone to create a sync schedule.
+
+:::info
+
+Only one sync per connection can run at a time. If cron schedules a sync to run before the last sync finishes, the sync will start after the last one completes. 
```

Comment:
@supertopher @davinchia Is this accurate?

Reply:
I believe this is accurate

Reply:
Sorry for the late comment.

I think we should also add that expressions lower than an hour are not accepted.

I'm not sure what the limits are on the OSS Airbyte deployments. I believe it is the same.

@mfsiega-airbyte can you confirm?

Reply:
In pseudocode, the logic here looks like: `next run start = nextScheduledRunAfter(previous run start + 60 seconds)`

So if you image that a cron expression "generates" a bunch of scheduled times, the scheduler picks the next one that's at least 60 seconds after the previous one.

There is no actual check on the expression itself apart from that it's a valid cron expression.

Code ref: https://github.com/airbytehq/airbyte/blob/master/airbyte-workers/src/main/java/io/airbyte/workers/temporal/scheduling/activities/ConfigFetchActivityImpl.java#L102

I don't have a strong opinion on whether this should change. It seems like there's maybe some utility in allowing users to effectively say "I want my sync to run ~continuously" to get some sorta-streaming behaviour; but that relies on the fact that we never run syncs concurrently which is a bit indirect.

If we do want to limit the cron granularity I think simplest would be to just say "the seconds/minutes have to be fixed". I think my vote would be allow once-per-minute and just say "the seconds have to be fixed", but again I don't feel so strongly.

@andyjih any thoughts on this?

Reply:
On the OSS side, I'm comfortable with letting users set any valid cron schedule they want.

That said, I know for Cloud, we removed the sub-1-hour schedule options to prevent oncall issues. @davinchia Do you think our cloud infra can handle schedules that are more frequent than 1-hour?

Reply:
@andyjih @davinchia 
Should I add a note saying something like "Cloud does not allow sub-one-hour syncs," or should I give Cloud users the option to contact support if they want to schedule sub-one-hour syncs?

Reply:
+1 to what Andy said.